### PR TITLE
Add GUI in preferences dialog for adding URI customized schemes

### PR DESCRIPTION
**Pull request by donho**
Related: https://github.com/notepad-plus-plus/notepad-plus-plus/commit/5168bdb7c5395821e458aaf20297cb7e8c76ef02 and https://github.com/notepad-plus-plus/notepad-plus-plus/issues/9155

### CODE REVIEW

File: PowerEditor/src/Parameters.h [hunk 1]
```diff
@@ -849,7 +849,7 @@ struct NppGUI final
 	bool _isWordCharDefault = true;
 	std::string _customWordChars;
 	urlMode _styleURL = urlUnderLineFg;
-	generic_string _customizedShemes = TEXT("svn:// cvs:// git:// imap:// irc:// irc6:// ircs:// ldap:// ldaps:// news: telnet:// gopher:// ssh:// sftp:// smb:// skype: snmp:// spotify: steam:// sms: slack:// chrome:// bitcoin:");
+	generic_string _uriShemes = TEXT("svn:// cvs:// git:// imap:// irc:// irc6:// ircs:// ldap:// ldaps:// news: telnet:// gopher:// ssh:// sftp:// smb:// skype: snmp:// spotify: steam:// sms: slack:// chrome:// bitcoin:");
```
Comment (sasumner) [inline PowerEditor/src/Parameters.h:852]: @donho Spelling `_uriShemes` -> `_uriSchemes` :-)

Comment (sasumner) [reply PowerEditor/src/Parameters.h:852]: But you didn't change the spelling problem?  :-)

**Comment by sasumner** [comment]
@donho

Okay, so I didn't actually review the code so much as try out the feature.

Taking a cue from [HERE](https://community.notepad-plus-plus.org/post/60031) (a user complaint about his custom linking behavior that used to work but is broken in 7.9.1), I recreated this user's setup on my system and tried to get it to work in the current (as of this writing) codebase.

I will report that I had success.

Summary:

I added this:
image

And after that I was able to achieve links in text such as these:
image
And then when clicking the links, the indicated files opened in Notepad++ for me.

I did notice that the links with pathnames must all contain forward slashes in order to be recognized as links, for example, this is NOT recognized as a link:
image

This is in contrast to earlier versions of N++ (I used 7.8.6) where backslashes WERE recognized, and mixed slashes as well, examples:
image

**Comment by donho** [comment]
@sasumner 
It's about @Uhf7 's implementation of URI. As he has mentioned, it's impossible to manage all the custom URI format, so it's based on URL format - I agree with his choice.

**Comment by sasumner** [comment]
>I agree with his choice.

I agree also.

**Pull request closed without merging**